### ISSUE TITLE
Installing seldon-core-operator requires clusterwide RBAC and should be installed by a cluster admin

### ISSUE THREAD

**Issue by nakfour** [SeldonIO/seldon-core#670]
Hi Everyone,

For Seldon release 0.3.1, it requires cluster wide permission to install. Previously, only installing the CRD required cluster wide permissions and we had an option to install seldon-core within a single namespace with admin rights only. Is it possible to get that option again with seldon-core-operator? 
Thanks

**Comment by cliveseldon** [comment]
We use kubebuilder and that is the default they provide. It also makes sense in that Custom Resources are clusterwide installs so you need cluster privileges to install them. However, I agree the running manager could be restricted to the namespace it is created it. 

We would need to check the defined way of allowing this in kubebuilder and if this is considered normal practice.

**Comment by cliveseldon** [comment]
This seems to be blocked by this open issue https://github.com/kubernetes-sigs/controller-tools/issues/225

**Comment by vpavlin** [comment]
Hi, the linked issue was fixed, would be willing to take a look at adding the option to use namespaced role?

**Comment by axsaucedo** [comment]
Closing in favour of #1139, which will only be available from later versions
